src/stores/url.store.ts

```diff
@@ -0,0 +1,19 @@
+import { defineStore } from 'pinia'
```

Comment:
Remove this store and maintain forwarder and rpcUrl in gateway service and export it from there

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/119/commits/a0b154b2272f25e889f7c83ab9a237d51398b217